packages/manager/modules/manager-components/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "@ovh-ux/manager-manager-components",
```

Comment:
```suggestion
  "name": "@ovh-ux/manager-components",
```

Reply:
The name of the module is `manager-components`. The prefix `@ovh-ux/manager-` gets added automatically

Reply:
Changed